.gitlab/bazel/include.yaml

```diff
@@ -0,0 +1,50 @@
+.bazel:defs:
+  cache:
+    - key:
+        prefix: bazel-$CI_RUNNER_DESCRIPTION
+        files: [ .bazelversion ]
+      paths:
+        - .cache/bazelisk
+        - .cache/bazel/install
+      policy: pull-push
+      when: on_success
+    - key: bazel-$CI_RUNNER_DESCRIPTION
+      paths:
+        - .cache/bazel/cache
```

Comment:
Do we really want to have this? CI Infra team is working on persistent runners this sprint so we should be able to reuse directories on the workers, sharing disk cache via Gitlab will actually add to the Runner's startup time